src/features/homePage/HomePage.tsx

```diff
@@ -1,5 +1,150 @@
+        />
+      </div>
+
+      <div>
+        <h2 className="ml-6 mb-9 text-2xl sm:mb-0">{t('subtitle')}</h2>
+      </div>
+      <div className=" relative mb-[6.3rem] flex max-w-7xl flex-wrap justify-center">
+        {mock
+          .sort(() => 0.5 - Math.random())
+          .slice(0, 6)
+          .map((index) => (
+            <div
+              key={index.id}
+              className="flex h-[27.8rem] w-[21.8rem] flex-col items-center py-0 sm:py-9"
+            >
+              <img
+                src={index.productImage}
+                alt={
+                  // index.altText
+                  i18n.language === 'en' ? index.altText.en : index.altText.pl
+
+                  // index.altText[i18n.language]
+                }
+                className="mb-20 h-52 w-52 rounded-full"
+              />
+              <h3 className="text-xl font-bold">{index.id}</h3>
+            </div>
+          ))}
+      </div>
+
+      <div className=" mx-6 mb-[3.1rem] flex max-w-7xl justify-between sm:px-3 md:px-6 lg:px-8">
+        <Formik
+          initialValues={{
+            email: '',
+            message: '',
+          }}
+          validationSchema={contactSchema}
+          onSubmit={(
+            values: IFormValues,
+            actions: FormikHelpers<IFormValues>,
+          ) => {
+            actions.setSubmitting(true);
+
+            emailjs
+              .send(
+                'service_ypr0zj8',
+                'template_z0d7tqj',
+                {
+                  email: values.email,
+                  message: values.message,
+                },
+                'user_pR6XzZUshqc9XuxuBLUzf',
+              )
+              .then(
+                () => {
+                  actions.setSubmitting(false);
+                  actions.resetForm();
+                },
+                () => {
+                  actions.setSubmitting(false);
+                },
+              );
+          }}
+        >
+          <div className="h-fit-content  w-full lg:w-[50%] ">
+            <h2 className="pb-7 text-2xl">{t('more')}</h2>
+            <h3 className="pb-7">{t('contactForm')}</h3>
+
+            <Form className="flex flex-col">
+              <Field
+                id="email"
+                name="email"
+                placeholder="Email"
+                className=" flex h-[3.75rem]  bg-[#F0F0F0] p-6 "
+              />
+              <ErrorMessage
+                component="a"
+                name="email"
+                className="mt-3 text-red-600"
+              />
+
+              <Field
+                name="message"
+                component="textarea"
+                placeholder={`${t('message2')}`}
+                className="mt-9 h-56 bg-[#F0F0F0] p-6 "
+              />
+
+              <ErrorMessage
+                component="a"
+                name="message"
+                className="mt-3 text-red-600"
+              />
+
+              <button
+                type="submit"
```

Comment:
Could we make the submit button the same height as the input for email maybe?

Reply:
done ✅